templates/bundles/TwigBundle/Exception/error404.html.twig

```diff
@@ -1,10 +1,9 @@
-{% extends 'visitor.html.twig' %}
+{% extends 'error.html.twig' %}
```

Comment:
non, c'est pas ça, c'était bien que ça étende visitor 
check la doc ! 
https://symfony.com/doc/current/controller/error_pages.html#overriding-the-default-error-templates

Reply:
je comprend pas bien la doc c'est quoi le principe